src/inference/src/system_conf.cpp

```diff
@@ -97,7 +101,8 @@ bool with_cpu_x86_avx512_core_amx_fp16() {
 }
 
 bool with_cpu_x86_avx512_core_amx() {
-    return with_cpu_x86_avx512_core_amx_int8() || with_cpu_x86_avx512_core_amx_bf16();
+    return with_cpu_x86_avx512_core_amx_int8() || with_cpu_x86_avx512_core_amx_bf16() ||
```

Comment:
We'd better not to change this function which is consistent with onednn.

Reply:
> We'd better not to change this function which is consistent with onednn.

recover it.